src/internal_rep.rs

```diff
@@ -248,6 +248,24 @@ impl Declared for TypeInfo {
 
 impl TypeInfo {
     pub fn new(td: TypeDecl, file: &SimpleFile<String, String>) -> Result<TypeInfo, CascadeErrors> {
+        let mut temp_vec = td.inherits.to_vec();
+        temp_vec.sort();
+        let mut iter = temp_vec.iter().peekable();
+        while let Some(cur_val) = iter.next() {
+            match iter.peek() {
```

Comment:
A match that only has one active branch is better as an `if let`.  (`if let Some(next_val) = iter.peek() { ... }`)

Reply:
(I suspect if you run clippy on this it will make this suggestion with the exact code to use).